src/utils.py

```diff
@@ -89,8 +89,9 @@ def return_hexadecimal(a: int) -> str:
     return hex(a)
 
 
-def return_random_number() -> int:
+def return_random_number() -> str:
```

Comment:
```suggestion
def return_random_number() -> float:
```
You made a small mistake here, it should be float I guess :)